src/client/src/app/settings/settings.component.ts

```diff
@@ -1,35 +1,64 @@
 import { Component, OnInit } from '@angular/core';
 import { HttpClient, HttpErrorResponse } from '@angular/common/http';
+import { HttpHeaders } from '@angular/common/http';
+import { IBoard } from '../../../../interfaces';
+import { httpPostOptions } from '../../../../config';
+import { catchError, retry } from 'rxjs/operators';
+import { Observable } from 'rxjs/Observable';
+import 'rxjs/add/operator/map';
+import 'rxjs/add/operator/catch';
+import { ReactiveFormsModule, FormsModule, FormGroup, FormControl, Validators } from '@angular/forms';
+
 
-interface IBoard {
-  title: string;
-  description;
-  imageURL;
-}
 @Component({
   selector: 'app-settings',
   templateUrl: './settings.component.html',
   styleUrls: ['./settings.component.css']
 })
 export class SettingsComponent implements OnInit {
-  currentBoard: IBoard = null; //new board with the attributes from the interface IBoard
-  id = "ID2";
-  requestURLHost: string = 'http://localhost';
-  requestURLPort: string = ':8888';
-  requestURLTag: string = '/title/';
-  requestURLID:string = '1';
-  requestURLBoard: string = this.requestURLHost+this.requestURLPort+this.requestURLTag+this.requestURLID;
-  length: string; 
-  
-
-  constructor(private http: HttpClient) { 
-  this.http.get(this.requestURLBoard).subscribe((data:IBoard) => {
-    this.currentBoard = data;
-
-      }
-    );
+
+  board = {
+    title: 'sword master'
+  };
+
+  requestHost = 'localhost';
+  requestPort = '34534';
+  requestLocation = 'title';
```

Comment:
Currently the request is done to the wrong location. Pls change 'title' to 'board' ..